modules/lib/lib-content/src/main/resources/lib/xp/examples/content/get.js

```diff
@@ -14,6 +14,34 @@ if (result) {
 }
 // END
 
+// BEGIN
+// Gets a single content by path and versionId.
+var result_by_path_and_version_id = contentLib.get({
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'result_by_path_and_version_id' is not in camel case. (camelcase)](https://app.codacy.com/app/enonic/xp/pullRequest?prid=4629451)